proposals-CG-WG.md

```diff
@@ -1,82 +1,78 @@
 
-  * Entrance criteria
-    * Nothing: no permission needed from anyone. 
-  * Entrance artifact
-    * An issue, a personal repo, a blog post, a tweet, etc 
-  * Exit criteria
+  * What's needed from individuals?
     * Identification of [champions](https://github.com/tc39/how-we-work/blob/main/champion.md)
-    * A problem statement 
-# Stage 1
+    * A good understanding of the Problem Space
+    * Optionally, Alternatives under consideration
+  * What's asked of the **Community Group**?
+    * The Communit Group consensus that the problem is worth spending the Community Group’s time working on
+  * What happens if accepted?
+    * Optionally, a repo gets created for the champions to develop the feature (e.g., [example](https://github.com/fedidcg/CrossSiteCookieAccessCredential)). Small features can also continue as issues.
 
-The purpose of Stage 1 proposals is to explore the Solution Space and pick a preferred direction. This is where most of the work gets done because it involves exploring alternatives, understanding tradeoffs, gathering implementation experience, incubating alternatives, gathering evidence of demand and fitness for purpose and finally identifying the best out of the many alternatives.
+# Stage 2: Formalization
 
-  * Entrance criteria
-    * Products of the prior stage and
-    * The CG consensus that the problem is worth spending the CG’s time working on
-  * Entrance artifact
-    * A FedID repo gets created in the CG for the champions to develop the feature (e.g. [example](https://github.com/fedidcg/CrossSiteCookieAccessCredential)).
-  * Exit criteria
+The purpose of Stage 2 Proposals is to formally specify the best (and seemingly workable) alternative that was identified in the prior step: handle corner cases, integrate with other parts, reconcile with other proposals, and resolve the concerns identified at the entrance. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.
+
+  * What's needed from champions?
     * An [explainer](https://tag.w3.org/explainers/)
-    * A specific proposal under consideration (with varying degrees of specificity, e.g. from code samples to a spec PR)
-    * Alternatives considered
-    * Known Issues filed
-    * Browser and developer implementation experience (e.g. a prototype, dev trials, origin trials, etc) 
+    * Alternatives and trade-offs considered
+    * A specific proposal with varying degrees of specificity required for the Working Group review, e.g., code samples, examples, or a spec PRs (if the formalism is needed to understand the proposal)
```

Comment:
The more I look at it, the more I think that "varying degrees of specificity" might be overspecifying the output.

Reply:
"A specific proposal with varying degrees of specificity required for the Working Group review, e.g., code samples, examples, or a spec PRs (if the formalism is needed to understand the proposal)"

Does this work?

Reply:
Resolved offline: @hlflanagan was looking at an old snapshot.